src/routes/blog/index.svelte

```diff
@@ -12,10 +28,23 @@
   import Spacing from '$lib/components/sections/spacing.svelte';
 
   export let posts: Newsletter[] = [];
```

Comment:
Expect there to be more properties passed down.
```suggestion
  export let posts: Newsletter[] = [];
  export let tags: string[] = [];
  export let selectedTags: string[] = [];
```

Reply:
Could you please explain more? Why do we need to `export` the variables `tag` & `selectedTags`?

Reply:
Now I feel like I begin to understand Svelte.